tests/integration/test_decision_worker.py

```diff
@@ -0,0 +1,148 @@
+"""
+Test worker process.
+"""
+
+import multiprocessing as mp
+import time
+
+from modules import drone_odometry_local
+from modules import odometry_and_time
+from modules.cluster_estimation import cluster_estimation
+from modules.decision import decision
+from modules.decision import decision_worker
+from utilities.workers import worker_controller
+from utilities.workers import queue_proxy_wrapper
+
+
+DISTANCE_SQUARED_THRESHOLD = 25  # Distance is in meters
+
+TOLERANCE = 0.1  # meters
+
+CAMERA_FOV_FORWARDS = 90  # degrees
+CAMERA_FOV_SIDEWAYS = 120  # degrees
+
+SEARCH_HEIGHT = 10  # meters
+SEARCH_OVERLAP = 0.2
+
+SMALL_ADJUSTMENT = 0.5  # meters
+
+WORK_COUNT = 5
+
+
+def simulate_cluster_estimation_worker(
+    min_activation_threshold: int,
+    min_new_points_to_run: int,
+    random_state: int,
+    input_queue: queue_proxy_wrapper.QueueProxyWrapper,
+) -> None:
+    """
+    Places list of detected landing pads into the queue.
+    """
+
+    result, estimator = cluster_estimation.ClusterEstimation.create(
+        min_activation_threshold, min_new_points_to_run, random_state
+    )
+    assert result
+    assert estimator is not None
+
+    result, cluster_pads = estimator.run(input_queue, False)
+    assert result
+    assert cluster_pads is not None
```

Comment:
When I run the test, it crashes here. However, that's less important.

Instead of instantiating the cluster estimation object, create a fake list of object in world that is placed into the queue, similar to `simulate_flight_interface_worker()` below. If you want to be extra fancy, you can optionally make a list of these so that they can be selected in the loop.

Test by running: `python -m tests.integration.test_decision_worker`

Reply:
What error are you getting within the cluster estimation function? When I try to run the test, I get a "ModuleNotFoundError".

Reply:
```
 if len(detections) == 0:
TypeError: object of type 'QueueProxyWrapper' has no len()
```